lib/dependencies.sh

```diff
@@ -31,14 +31,16 @@ run_if_present() {
 
 yarn_node_modules() {
   local build_dir=${1:-}
+  $NODE_VERBOSE && local opt_verbose='--verbose' || local opt_verbose=''
```

Comment:
Whenever an assignment is definitely happening, I tend to like using something like a ternary instead of a conditional because it does a better job of conveying intent. "We're creating and assigning `opt_verbose` and you can use it after this." So in JS I would use `const optVerbose = process.env.verbose ? '--verbose' : ''` and be pretty happy with that.

But I agree this is quite ugly in bash. Is there a better way to do it? If not I'm fine changing it to an if.